src/LinAlg/SparseMatrix.C

```diff
@@ -1272,6 +1273,7 @@ bool SparseMatrix::solveSLUx (Vector& B, Real* rcond)
     if (rcond && slu)
       *rcond = slu->rcond;
   }
+#endif
 
   return true;
```

Comment:
Ok, this was due to the refactoring I did in b32d2cd9d159dab424099d6fe54ffbda64de4ad8. Another "error" introduced there is that this method is supposed to return `false` when no superlu, i.e., L178 should be replaced by
```
return ierr == 0;
```

Reply:
ok.